lib/checks/media/frame-tested.json

```diff
@@ -0,0 +1,15 @@
+{
+  "id": "frame-tested",
+  "evaluate": "frame-tested.js",
+  "options": {
+    "isViolation": false
+  },
+  "metadata": {
+    "impact": "critical",
+    "messages": {
+      "pass": "The iframe can be tested with axe-core",
```

Comment:
This should probably be "The iframe _was_ tested with axe-core" so as to position it opposite the failure message. As-is, it suggests that it's now safe to test.